tools/api-generator/src/main/java/com/microsoft/playwright/tools/ApiGenerator.java

```diff
@@ -127,15 +118,38 @@ private static String formatSpec(JsonArray spec) {
           }
         }
         out.add("<li> " + beautify(text) + "</li>");
+        continue;
       } else {
         if (currentItemList != null) {
           out.add("</" + currentItemList + ">");
           currentItemList = null;
         }
+      }
+      if ("code".equals(type)) {
+        if (!node.get("codeLang").getAsString().contains("java")) {
+          continue;
+        }
+        out.add("<pre>{@code");
+        for (JsonElement line : node.getAsJsonArray("lines")) {
+          out.add(line.getAsString());
+          if ("client.send(\"Animation.setPlaybackRate\", params);".equals(line.getAsString())) {
+            System.out.println(line);
+          }
+        }
+        out.add("}</pre>");
+      } else {
         String paragraph = node.get("text").getAsString();
-        paragraph = beautify(paragraph);
-        if ("note".equals(type)) {
-          paragraph = "<strong>NOTE:</strong> " + paragraph;
+        Matcher matcher = Pattern.compile("\\*\\*(.+)\\*\\*").matcher(paragraph);
```

Comment:
Should this be `^....$`? Otherwise you seem to miss prefix/suffix at line 147.

Reply:
`matcher.matches()` already matches full string. But I can change the pattern for clarity too.